README.md

```diff
@@ -7,367 +7,506 @@
+`TimeInstant` reported for each measurement will be the hour when that
+measurement was observed (e.g. 4:00 PM), while all the measurements will have an
+arrival time around midnight. If no timestamps were reported with such
+measurements, the `TimeInstant` attribute would take those values around
+midnight.
 
 This functionality can be turned on and off through the use of the `timestamp` configuration flag (described in the
 configuration), as well as 'timestamp' flag in device or group provision.
 
+
 ### Implementation decisions
-Given the aforementioned requirements, there are some aspects of the implementation that were chosen, and are
-particularly under consideration:
-* Aside from its configuration, the IoT Agent Lib is considered to be stateless. To be precise, the library
-mantains a state (the list of entities/devices whose information the agent can provide) but that state is considered
-to be transient. It's up to the particular implementation of the agent to consider whether it should have a persistent
-storage to hold the device information (so the internal list of devices is read from a DB) or to register the devices
-each time a device sends a measure. To this extent, two flavours of the Device Registry has been provided: a transient
-one (In-memory Registry) and a persistent one (based in MongoDB).
-* The IoT Agent does not care about the origin of the data, its type or structure. The mapping from raw data to the
-entity model, if there is any, is a responsability of the particular IoT Agent implementation, or of another third
-party library.
 
-##  Build and Install
+Given the aforementioned requirements, there are some aspects of the
+implementation that were chosen, and are particularly under consideration:
+
+-   Aside from its configuration, the IoT Agent Lib is considered to be
+    stateless. To be precise, the library mantains a state (the list of
+    entities/devices whose information the agent can provide) but that state is
+    considered to be transient. It's up to the particular implementation of the
+    agent to consider whether it should have a persistent storage to hold the
+    device information (so the internal list of devices is read from a DB) or to
+    register the devices each time a device sends a measure. To this extent, two
+    flavours of the Device Registry has been provided: a transient one
+    (In-memory Registry) and a persistent one (based in MongoDB).
+-   The IoT Agent does not care about the origin of the data, its type or
+    structure. The mapping from raw data to the entity model, if there is any,
+    is a responsability of the particular IoT Agent implementation, or of
+    another third party library.
+
+## Install
+
+Information about how to configure the Library can be found at the corresponding
+section of the
+[Installation & Administration Guide](https://iotagent-node-lib.readthedocs.io/en/latest/installationguide).
```

Comment:
We should avoid cross-doc linking. I mean, MD -> MD and RTD -> RTD is ok, but MD -> RTD should be avoided.

Note the user could be checking this doc in a given tag (e.g. tag 2.8.0), so `doc/installationguide.md` would correctly point to the a document in the same tag. However, if you link to https://iotagent-node-lib.readthedocs.io/en/latest/installationguide that document correspond to master, not 2.8.0.

Reply:
Fixed -  e12947e - I still don't like this practice though.

You and I have a different opinion on this - I find RTD documentation much easier to navigate and therefore would like to promote RTD as the primary source.

I view these links as links to other useful resources, not navigation to other markdown files.
For example, for the other IoT Agents, I may expect to find a  link to `https://iotagent-node-lib.readthedocs.io` rather than  `https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/index.md` (or should that be to a specific hash?)

In my experience, when promoting documentation, it is standard practice to link to the **latest** version rather than legacy. (especially as this encourages to people to upgrade)